src/frontend/app/features/cloud-foundry/edit-organization/edit-organization-step/edit-organization-step.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="edit-org">
+  <div>Specify the new name for the organization (leave unchanged if don't wish to rename it): </div>
+  <form [formGroup]="editOrgName" class="edit-org__stepper-form">
+    <mat-form-field>
+      <input matInput formControlName="orgName" [(ngModel)]="orgName" placeholder="Organization Name">
+    </mat-form-field>
+    <div class="edit-org__stepper-form__ssh">
+      <div>
+        Change Organization Status ( Currently {{ currentStatus }})
+      </div>
+      <div>
+        <mat-slide-toggle formControlName="toggleStatus" [(ngModel)]="status">
```

Comment:
If we toggle this, does the console treat the org correctly? If not, we should hide it.